cijoe/configs/verify-phys-linux.toml

```diff
@@ -0,0 +1,114 @@
+[cijoe.transport.ssh]
+hostname = "verify-debian.home.arpa"
+port = 22
+username = "root"
+password = "root"
+
+[os]
+name = "debian"
+version = "bookworm"
+
+[liburing.repository]
+remote = "https://github.com/axboe/liburing.git"
+path = "/root/git/liburing"
+tag = "liburing-2.4"
+
+[xnvme.repository]
+remote = "https://github.com/xnvme/xnvme.git"
+path = "/root/git/xnvme"
+branch = "next"
+
+# This is utilized by repository syncing during development.
```

Comment:
It would be nice if our CI configs didn't include anything not needed for CI.
My feeling is that a lot of these lines only relate to development

Reply:
I think the comment was misleading. The `[xnvme.repository.sync]` didn't need `remote_alias` and `branch`, so I've removed them, but the remote path is used when provisioning using tgz.